### PR TITLE
Completed and ready for review issue-3-user-can-edit-an-entry.

### PR DESCRIPTION
**Pull request by Mitchelllund**
Code-Journal-#3

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -6,28 +6,45 @@ var $entriesPage = document.querySelector('#entries-page');
...
+  } else if (data.editing !== null) {
+    var updatedValue = {};
+    updatedValue.title = $formId.title.value;
+    updatedValue.photoUrl = $formId.photoUrl.value;
+    updatedValue.notes = $formId.notes.value;
+    updatedValue.nextEntryId = data.editing.nextEntryId;
+    data.entries.splice(data.entries.length - data.editing.nextEntryId, 1, updatedValue);
+    $entriesPage.classList.remove('hidden');
+    $entryForm.classList.add('hidden');
+    var entryLocation = $li.item(data.entries.length - data.editing.nextEntryId);
```
Comment (BrettAlbright) [inline js/main.js]: this is what you want to change. Once you start deleting entries, the numbers won't align in this way. Instead, you want to loop through all of the `li` elements and find the one with the matching entryid so you know which one to replace.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Great work Mitch! Small update needed!

**Comment by Mitchelllund** [comment]
Code-Journal-#3 updates

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**